### PR TITLE
add Pusher Channels mention where absent

### PR DESCRIPTION
**Pull request by annzenkina**

### CODE REVIEW

**Comment by annzenkina** [comment]
@WillSewell done. Not entirely sure, but I think we might need to update the docs here as well: https://www.npmjs.com/package/pusher. What do you think?

**Comment by WillSewell** [comment]
> I think we might need to update the docs here as well: https://www.npmjs.com/package/pusher

I think these will be picked up automatically from the master of Github.

File: package.json [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "name": "pusher",
-  "description": "Node.js client to interact with the Pusher REST API",
-  "version": "2.2.0",
+  "description": "Node.js client to interact with the Pusher Channels REST API",
+  "version": "2.2.1",
```
Comment (WillSewell) [inline package.json]: I'm not sure you actually need to make a release for these changes to be picked up on https://www.npmjs.com/.

Comment (annzenkina) [reply package.json]: hm. I'll try to merge the PR without changing the version and see how it goes.

**Pull request merged**